src/components/Providers.tsx

```diff
@@ -62,12 +70,12 @@ export const arbitrumSepolia: Chain = {
   network: 'arbitrum-sepolia',
   nativeCurrency: ether,
   rpcUrls: {
-    default: rpcURLs[ChainId.ArbitrumSepolia]!,
-    public: rpcURLs[ChainId.ArbitrumSepolia]!,
+    default: { http: [rpcURLs[ChainId.ArbitrumSepolia]!] },
+    public: { http: [rpcURLs[ChainId.ArbitrumSepolia]!] },
   },
 };
 
-const { provider, webSocketProvider } = configureChains(
+const { provider, chains } = configureChains(
```

Comment:
for some reason i managed to connect via WalletConnect (Sepolia Safe) the first time, but then WC wouldn't show me a QR code very time I tried.... I am not sure what's happening??

<img width="936" alt="image" src="https://github.com/OffchainLabs/retryable-tx-panel/assets/13184582/f02f08be-019e-473f-ae51-ef0d15a84c42">

Reply:
I think it's the same issue as the bridge, I'll clear the localStorage on first load